### ISSUE TITLE
6.5.4 Support

### ISSUE THREAD

**Issue by Grunticus03** [aaronoah/kibana_calendar_vis#1]
Any plans to update this to be compatible with 6.5.4?

**Comment by aaronoah** [comment]
yes, I'm recently considering upgrade it. There is no specific time table for it to be released though.

**Comment by karthikgreentrend** [comment]
does it work for 6.5.1 as well ?

**Comment by aaronoah** [comment]
Unfortunately, you have to modify the version number on package.json for it to match Kibana version. I will later ship releases compatible up to that.

**Comment by karthikgreentrend** [comment]
thanks @aaronoah
